### DISCUSSION TITLE
.Net 7 Maui Support has ended

### DISCUSSION THREAD

**Discussion by JRosanowski** [CommunityToolkit/Maui#2157] [Q&A] [answered]
.Net Maui 7.0 support ended on 14 May 2024. Are there any plans to move to .Net Maui 8.0?

https://dotnet.microsoft.com/en-us/platform/support/policy/maui

**Comment by bijington** [comment, marked as answer]
We have supported .NET 8.0 since it was released in November 2023.

**Reply by JRosanowski** [reply]
The nuget still says the dependencies are net7.0, net7.0-android33 etc or am I looking at the wrong thing?

https://www.nuget.org/packages/Trimble.Modus.Components

**Reply by bijington** [reply]
You are definitely looking at the wrong thing... If you click the `Project website` link on that page you linked it will take you to a different GitHub repository (https://github.com/trimble-oss/modus-mobile-maui-components). That being said while it targets net7.0 it will technically support net8.0 short of dealing with any breaking changes between the 2 versions.

If you are using this toolkit then it is this package: https://www.nuget.org/packages/CommunityToolkit.Maui among some others but you will see we only target net8.0.

**Reply by JRosanowski** [reply]
Sorry, this is a result of too many tabs open. :(

**Comment by CliffAgius** [comment]
Closed as Answered.
